fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBMetaDataStore.java

```diff
@@ -234,7 +241,17 @@ protected RecordMetaData buildMetaData(RecordMetaDataProto.MetaData metaDataProt
      * @return a future that completes when the save is done
      */
     public CompletableFuture<Void> saveAndSetCurrent(@Nonnull RecordMetaDataProto.MetaData metaDataProto) {
-        RecordMetaData validatedMetaData = buildMetaData(metaDataProto, true);
+        return saveAndSetCurrent(metaDataProto, null);
+    }
+
+    /**
+     * Build meta-data to use from Protobuf and save.
```

Comment:
Might be worth including a link to the `setLocalFileDescriptor` method in `RecordMetaDataBuilder` and including something briefly about what a "local file descriptor" is, because that isn't necessarily obvious to a user who arrives at this function.